ang/civicase/ContactCard.js

```diff
@@ -31,6 +31,16 @@
         $document.bind('click', closeDropdownIfClickedOutside);
       }());
 
+      /**
```

Comment:
Shouldn't these be declared inside controller function and not link function. I wonder why other scope related function are a part of the link function and not controller function. Can we move the functions which are not DOM related to the controller function of this directive ?